### PR TITLE
feat(cli): add generate command for cli

### PR DESCRIPTION
**Pull request by mttrbrts**
Adds a new command to the Concerto command line tool to generate a sample JSON instance of a type.

```console
$ concerto generate sample --model person.cto --concept org.accordproject.person.Person --includeOptionalFields
{
  "$class": "org.accordproject.person.Person",
  "identifier": "resource1",
  "additionalName": "Duis in ut ad elit.",
  "address": {
    "$class": "org.accordproject.address.PostalAddress",
    "streetAddress": "Officia.",
    "postalCode": "Reprehenderit aliqua pariatur.",
    [...]
}
```

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `master` from `fork:branchname`

### CODE REVIEW

**Comment by mttrbrts** [comment]
FYI @stefanblaginov

File: packages/concerto-cli/index.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ require('yargs')
                 .then((result) => {
                     Logger.info('Input is valid');
                     if (!options.functional) {
-                        Logger.info(result);
+                        console.log(result);
```
Comment (dselman) [inline packages/concerto-cli/index.js:75]: Intentional?

File: packages/concerto-cli/index.js [hunk 2]
```diff
@@ -215,7 +215,7 @@ require('yargs')
         return Commands.parse(argv.model, argv.resolve, argv.all, argv.output, options)
             .then((result) => {
                 if (result) {
-                    Logger.info(result);
+                    console.log(result);
```
Comment (dselman) [inline packages/concerto-cli/index.js:218]: Intentional?

**Review by dselman — CHANGES_REQUESTED** [review]

File: packages/concerto-cli/index.js (hunk 1 above)
Comment (mttrbrts) [reply packages/concerto-cli/index.js:75]: Yes, I should have noted this as an unrelated change. `Logger` adds a log of formatting and metadata which makes the output difficult to pipe to another command or to stream to a file.

I made this decision for the new `generate` command, then decided to standardise the output of similar commands.

File: packages/concerto-cli/index.js (hunk 2 above)
Comment (mttrbrts) [reply packages/concerto-cli/index.js:218]: As above.

**Review by dselman — APPROVED** [review]

**Pull request merged**